app/controllers/admin/merchants_controller.rb

```diff
@@ -2,4 +2,16 @@ class Admin::MerchantsController < ApplicationController
   def show
     @merchant = Merchant.find(params[:id])
   end
+
+  def update
+    merchant = Merchant.find(params[:id])
+    if merchant.enabled == true
+      merchant.update(enabled: !merchant.enabled)
```

Comment:
@Jonathan-M-Wilson why not `merchant.update(enabled: false)` ? The way it's written now is a little confusing..?

Reply:
You are correct! Good advice! I will go back and change these!